src/app/modules/main/profile_section/contacts/controller.nim

```diff
@@ -180,3 +197,12 @@ proc shareUserUrlWithChatKey*(self: Controller, pubkey: string): string =
 
 proc shareUserUrlWithENS*(self: Controller, pubkey: string): string =
   self.contactsService.shareUserUrlWithENS(pubkey)
+
+proc requestProfileShowcaseForContact*(self: Controller, contactId: string, validated: bool) =
+  self.contactsService.requestProfileShowcaseForContact(contactId, validated)
+
+proc fetchProfileShowcaseAccountsByAddress*(self: Controller, address: string) =
+  self.contactsService.fetchProfileShowcaseAccountsByAddress(address)
+
+proc getEnabledChainIds*(self: Controller): seq[int] =
+  return self.networkService.getNetworks().filter(n => n.enabled).map(n => n.chainId)
```

Comment:
@MishkaRogachev had a conflict here because on master we're using a new API for the networks. Solved using `getNetworks` instead of `getCurrentNetworks`

Reply:
yea, getNetworks should be fine